sdk/core/azure-core/README.md

```diff
@@ -22,14 +22,16 @@ as an example of how to use the shared azure-core components in your client libr
   APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
 * You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
 
+### Versions available
```

Comment:
The H3 `Versions available` says "Versions", but we point only one, i.e. the latest version

Reply:
That is a little confusing, agreed. I believe the template requires a "Versions available" header, but we don't have an easy way to link to a filtered list of all releases of just this library. We could do what I did in iOS and just say. "Older versions are also available." with a link to the GH release page. What do you think?

Reply:
ah, was not aware of template requirement. I'm good at following the same iOS did.